src/main/groovy/grails/plugin/sentry/SentryConfig.groovy

```diff
@@ -33,11 +40,10 @@ import groovy.transform.CompileStatic
             LOW: [java.io]
  */
 
-@CompileStatic
+@CompileStatic(TypeCheckingMode.SKIP)
```

Comment:
As far as I understand [the docs](http://docs.groovy-lang.org/next/html/gapi/groovy/transform/CompileStatic.html), this essentially disables CompileStatic on the whole class.

So either remove the annotation, or fix the compile errors.

Reply:
You are right. It makes sense for overriding class `@CompileStatic` on method level, but it's redundant on class level. I will remove it.

Reply:
Btw @CompileStatic here causes groovy incompatibility with our project.

Reply:
054bf4ba